### PR TITLE
[DOC] classification notebooks upgrade

### LINKED ISSUE #1183: [DOC] Formatting of references wrong for docs on hybrid classifiers
### Describe the issue linked to the documentation

formatting at the end of this page for references is wrong
https://www.aeon-toolkit.org/en/stable/examples/classification/hybrid.html

the notebook that generates this is here

https://github.com/aeon-toolkit/aeon/blob/main/examples/classification/hybrid.ipynb

Need to add line breaks and link text to references (see other notebooks).

### Suggest a potential alternative/fix

_No response_

### PR DESCRIPTION
**Pull request by TonyBagnall**
fixes #1183 
see #1181 

added performance comparisons for classifiers by type. Currently loading from tsc.com, will probably put the results in benchmarking/example_results when done.  

Also fixed some typos/formatting things.

### CODE REVIEW

**Review by hadifawaz1999 — DISMISSED** [review]
LGTM

File: examples/classification/convolution_based.ipynb [hunk 1]
```diff
@@ -246,6 +246,137 @@
     "`n_features_per_kernel` are retained.\n"
```
Comment (MatthewMiddlehurst) [inline examples/classification/convolution_based.ipynb:246]: <p>Do we want to add the other Rocket configs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aeon-toolkit/aeon/pull/1185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

File: examples/classification/deep_learning.ipynb [hunk 2]
```diff
@@ -23,14 +23,14 @@
   },
```
Comment (MatthewMiddlehurst) [inline examples/classification/deep_learning.ipynb:23]: <pre class="ql-syntax" spellcheck="false">Line #5.    # Not doen these yet
</pre><p>Typo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aeon-toolkit/aeon/pull/1185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: examples/classification/distance_based.ipynb [hunk 3]
```diff
@@ -83,7 +83,7 @@
     "## Distance based classifiers\n",
```
Comment (MatthewMiddlehurst) [inline examples/classification/distance_based.ipynb:83]: <p>PF and EE are Java right? Typo here also</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aeon-toolkit/aeon/pull/1185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

File: examples/classification/hybrid.ipynb [hunk 4]
```diff
@@ -154,25 +154,153 @@
     "collapsed": false
```
Comment (MatthewMiddlehurst) [inline examples/classification/hybrid.ipynb:154]: <p>TS-CHIEF comment is wrong, also these are Java results</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aeon-toolkit/aeon/pull/1185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

**Review by MatthewMiddlehurst — APPROVED** [review]

**Pull request merged**